basil/HL/sensirion_sht85.py

```diff
@@ -0,0 +1,179 @@
+# ------------------------------------------------------------
+# Copyright (c) All rights reserved
+# SiLab, Institute of Physics, University of Bonn
+# ------------------------------------------------------------
+#
+
+import numpy as np
+import logging
+
+from basil.HL.SensirionBridgeDevice import SensirionBridgeI2CDevice
+
+logger = logging.getLogger(__name__)
+
+class sensirionSHT85(SensirionBridgeI2CDevice):
+    '''
+    Driver for the Sensirion SHT85 temperature and humidity sensor. 
+    Measurements can be performed in three repeatability modes:
+    low (0.15°C 0.21%RH) (default), medium (0.08°C 0.15%RH), high (0.04°C 0.08%RH)
+    with respective drawbacks in readout speed and power cosumption.
+    The dew point can be estimated using humidity and temperature.
+    Readout may also be performed asynchronously: 
+    The sensor can measure periodicly with 0.5, 1, 2, 4 or 10 measurements per second.
+    A non-blocking call to the asynchrounous read function may return None if no data is available.
+    In ART mode the humidity response is accelerated.
+    A heater may be turned on for debugging purposes.
+    '''
+
+    def __init__(self, intf, conf):
+        super(sensirionSHT85, self).__init__(intf, conf)
+
+    def init(self):
+        super(sensirionSHT85, self).init(0x44)
+
+        try:
+            import crcmod
```

Comment:
why import in `init`?

Reply:
the you can use a yaml describing your setup (even with this sensor in the yaml), that works as long as you do not call init. Fine with me here